tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -116,10 +129,8 @@ def test_positive_validate_katello_certs_check_output(self):
         """
         with get_connection() as connection:
             result = connection.run(
-                'katello-certs-check -c /tmp/{0} -k /tmp/{1} '
-                '-b /tmp/{2}'.format(
-                    self.cert_file_name, self.key_file_name, self.ca_bundle_file_name
-                ),
+                'katello-certs-check -c {} -k {} '
+                '-b {}'.format(self.cert_file_name, self.key_file_name, self.ca_bundle_file_name,),
```

Comment:
the string on this line can be brought up to the one above. you could put everything on line 132 and let black format it.

Reply:
OK